cli/cli-utils/src/packageIsInstallable.ts

```diff
@@ -21,23 +21,27 @@ export function packageIsInstallable (
   }
 ) {
   const pnpmVersion = packageManager.name === 'pnpm'
```

Comment:
This variable is used below too. I am not sure it should be changed for that code too.

Reply:
Replied in the comment below as it turned out this PR fixed another bug.